enterprise/backend/src/metabase_enterprise/sso/integrations/sso_utils.clj

```diff
@@ -34,6 +35,8 @@
   [user :- UserAttributes]
   (u/prog1 (first (t2/insert-returning-instances! User (merge user {:password (str (random-uuid))})))
     (log/info (trs "New SSO user created: {0} ({1})" (:common_name <>) (:email <>)))
+    ;; publish user-invited event for audit logging
+    (events/publish-event! :event/user-invited (assoc <> :sso_source (:sso_source user)))
```

Comment:
Hmm.. fair point. I think I'll leave this as-is for now, because we don't seem to need :sso_source much throughout the app, so I'll leave default fields alone. 

I can open a new PR to revisit this if we need to, but I think this approach is ok.